src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```diff
@@ -127,79 +98,28 @@ internal async Task InitializeRpcServerAsync()
             }
             catch (Exception grpcInitEx)
             {
-                var hostInitEx = new HostInitializationException($"Failed to start Rpc Server. Check if your app is hitting connection limits.", grpcInitEx);
+                throw new HostInitializationException($"Failed to start Rpc Server. Check if your app is hitting connection limits.", grpcInitEx);
             }
         }
 
         internal Task InitializeChannelsAsync()
         {
-            // TODO: Remove special casing when resolving https://github.com/Azure/azure-functions-host/issues/4534
-            if (ShouldStartAsPlaceholderPool())
-            {
-                return _webHostRpcWorkerChannelManager.InitializeChannelAsync(_workerRuntime);
-            }
-            else if (ShouldStartStandbyPlaceholderChannels())
+            if (_placeholderLanguageWorkersList == null)
             {
-                return InitializePlaceholderChannelsAsync();
+                throw new ArgumentNullException(nameof(_placeholderLanguageWorkersList));
             }
 
-            return InitializeWebHostRuntimeChannelsAsync();
-        }
-
-        private Task InitializePlaceholderChannelsAsync()
-        {
-            if (_environment.IsLinuxAzureManagedHosting())
+            if (_environment.IsPlaceholderModeEnabled())
             {
-                return InitializePlaceholderChannelsAsync(OSPlatform.Linux);
-            }
+                _logger.LogDebug("Initializing language worker channels '{placholderChannelList}' in placeholder mode.", string.Join(",", _placeholderLanguageWorkersList));
```

Comment:
typo `placholderChannelList` -> `placeholderChannelList`

Reply:
if we can have the worker_runtime logged in this line would make writing queries easier. alert can validate
if runtime=dotnet=>  dotnet,python,node are started
node => only node is started , ..

Reply:
done!